### ISSUE TITLE
Make always-visible block icon accessible and usable

### ISSUE THREAD

**Issue by chrisvanpatten** [WordPress/gutenberg#11669]
In #11600, we made the UI always show the block icon.

<img width="553" alt="48197149-ba743e00-e34c-11e8-8d83-581bceffe379" src="https://user-images.githubusercontent.com/1231306/48264543-d5e85300-e3f7-11e8-8afb-9ae7169409b8.png">

However it's being displayed in a low opacity state, with the idea being that it's not an active button like the block type switcher.

However, I would argue that in this case, the icon is serving an **entirely different purpose than the block switcher button**: it's a _visual label_, which helps indicate/reinforce the current block.

Removing the dropdown arrow changes the context of the icon entirely. Lowering the opacity, as a way of further reinforcing a disabled state, makes the icon inaccessible as a visual aid for indicating the current block.

Further, I would argue that it sends a confusing message. The dimmed block icon can be easily interpreted as a reflection of the block's own state, suggesting the block _itself_ is disabled.

I have a few potential ideas:

1. Simply show the block icon at full opacity, without the dropdown icon
2. Include the full switcher, and make the _switcher_ icon appear visually disabled when hovered
3. Continue using the low opacity but still include the dropdown arrow so it's clearer that it's the dropdown UI that's disabled, not the block in some way
4. Separate the block icon and switcher into two toolbar buttons, only showing the switcher icon when it has available options, and always showing the block icon at full opacity

Fundamentally, what's the point of always showing the icon if a certain subset of users can't see it?

**Comment by afercia** [comment]
> Fundamentally, what's the point of always showing the icon if a certain subset of users can't see it?

Good question.

**Comment by afercia** [comment]
Removing the "Needs Accessibility Feedback" label, as feedback is implied in the comment above.

**Comment by afercia** [comment]
I'd agree it should be purely decorative, e.g. a `<span>` and it shouldn't be wrapped in a `<Toolbar>` component. 

One more problem with this disabled button is that it uses an aria-label `Block icon` which doesn't tell anything meaningful to users:

![screenshot 2018-12-01 at 14 57 15](https://user-images.githubusercontent.com/1682452/49329025-b7e6bc00-f579-11e8-9f77-dcbeef187dda.png)

(reminder: when using a screen reader it is possible to navigate through all content, not only focusable elements)

**Comment by timwright12** [comment]
I can probably tackle this later in the week (via @10up)

**Comment by jasmussen** [comment]
> Curious if you have any ideas to iterate this to improve visibility under different conditions? 

The opacity is due to it being disabled. But we can consider a different disabled state here, perhaps where the opacity of the icon is higher and monochrome (i.e. desaturate all icons), and with a light gray background?

**Comment by chrisvanpatten** [comment]
@jasmussen Yeah, I think that would be helpful. I understand the intent but since block authors can change icon colors it can lead to situations where we can't guarantee the icon is appropriately visible (e.g. a low opacity black will be very different from grey at the same low opacity).

Taking control of the icon color in disabled states would be a great way to overcome this problem.

**Comment by mapk** [comment]
Looks like this is fixed with https://github.com/WordPress/gutenberg/pull/13721 so closing. If I missed anything, we can always reopen it.
